desktop/core/src/desktop/middleware_test.py

```diff
@@ -233,3 +234,31 @@ def dummy_get_response(request):
   finally:
     reset()
 
+def get_response(request):
+  return request
+
+class TestMultipleProxyMiddleware(TestCase):
```

Comment:
Nit: I think no need to import `TestCase`, we can give `object` or `unittest.TestCase` from unittest lib.

Also below we can user `assert_equal` from nose tools instead of `self.assertEqual`

Reply:
Yes true. using `unittest.Testcase`